### DISCUSSION TITLE
Cablecard vs. OTA on the same system, priority?

### DISCUSSION THREAD

**Discussion by KryptoNyte-Sage** [OpenSageTV/Community#36] [Guide/EPG]
My Sage system originally has cablecard (Ceton 6 pcie) with 2 of those tuners used for copy-freely (local affiliates), and 4 used for copy-once (WMC Live Tuners) non-locals.  I prefer to have my recordings with direct stream video and audio (5.1 audio if possible) so the cablecard copy-freely fits that bill, but it's only 2 tuners.  I had an older HDHR Quatro laying around, so I hooked it up today and added those (4) OTA tuners to Sage.  The 2 Ceton tuners are priority 10 and I added the Quatro tuners as priority 8, and have rebooted the systems numerous times since verifying the priority in the UI tuner setting.

Before adding the Quatro, my local affiliates coming from the cablecard were 1004, 1006, 1010, 1012, etc.  When I added the Quatro, SageTV [very intelligently] merged tuner guides and decided that 1004 would now show in the guide as 4.1 (OTA channel), 1006 as 6.1, etc.  I'd prefer to get this back to the cablecard channel numbers, and I feel like there must be a method or switch to do this.  The current result when channel 4.1 is tuned is Sage uses the cablecard on channel 1004 (with 1004 never showing in the guide).

How do I reverse this condition?  Is it based on which tuner was added last, in this case the Quatro OTA?

**Comment by Ugadata** [comment]
remap the logical channel number(s) on the OTA line to what you want it to be.

**Reply by KryptoNyte-Sage** [reply]
Thanks.  I was kind of hoping there would be a native way for one tuner to override the other so I wouldn't have to 'fake' it.  When new channels are added, it turns into a constant stream of maintenance.

**Reply by eelpout** [reply]
yeah, there is a way. in the Sage.properties file look over all your encoders/tuners (they'll start with mmc/encoders). the ones with the higher "encoder_merit" value will be used first. like this:

_mmc/encoders/-xxxxxxxx/encoder_merit=4_

make sure the SageTV server process is shut down before editing the file.

**Reply by andyvshr** [reply]
BTW: if the Extra options are enabled in Detailed Setup -> customize, there is an option near the bottom to be able to edit those merit values in the UI for tuner setup.

**Reply by eelpout** [reply]
heh, I forgot there was that option added for merits in the UI, so used to doing everything in the file. :)

**Reply by KryptoNyte-Sage** [reply]
Thanks for the replies.  When I say "priority" in my original post, I mean "merit."  The tuners are already set with the 1000+ channel tuners as the highest merit.
